.github/workflows/e2e-external.yml

```diff
@@ -0,0 +1,56 @@
+name: End-to-end tests for external PRs
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+concurrency:
+  group: ${{ github.event_name }}-${{ github.ref }}
+
+jobs:
+  test:
+    name: Run end to end tests
+    # If reviewed by a repo(/org) owner
+    if: |
+      github.event.pull_request.author_association != 'MEMBER'
+      && github.event.review.author_association == 'MEMBER'
+      && github.event.review.state == 'approved'
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: write
```

Comment:
```suggestion
```

Reply:
If you approve, and then the test fail, and then I approve, I believe the code you suggested will change the result to "Ok", because the tests won't run, but the steps will be succesful.